web/pf4/src/components/ChartWithLegend.tsx

```diff
@@ -13,6 +13,7 @@ type Props = {
   data: VCLines;
   seriesComponent: React.ReactElement;
   unit: string;
+  xUnit?: string;
```

Comment:
As per my previous comment, if you want to let caller control the ticks format, I'd rather like you expose here a props `xTicksFormat` function, then pass it down to `ChartAxis`, instead of this `xUnit` field. It will give more control to the caller.